### DISCUSSION TITLE
Shopify CLI and ThemeKit

### DISCUSSION THREAD

**Discussion by DanPete** [uicrooks/shopify-theme-lab#94] [Show and tell]
Really just wanted to make a quick post about how great this repo is. I love how easy it is to customize. 

I got worried last night that I might not be able to use this build tool with Shopify CLI since it seems like they are going to depreciate ThemeKit, but I really just have to change around the npm scripts a bit and got it working with Shopify CLI pretty nicely.

Thanks for the great work!  

Haven't figured it out perfectly yet, but just to get it set up quickly, I run `shopify theme serve` in the `shopify` directory and then just run `npm run dev` in another terminal tab so it's only doing the webpack dev config. And Shopify CLI is handling the reloading quite nicely now by itself.

**Comment by sergejcodes** [comment]
Hi, thanks!

Shopify has a tendency to shuffle its tools every now and then. Their CLI got updated just recently I'll have to look into it.

 I think you could be able to start the serve and dev tasks with only one command (from the root directory) by adjusting your scripts like this:

```json
{
  "scripts": {
    "start": "npm-run-all --parallel --silent --race dev shopify:serve",
    "dev": "cross-env NODE_ENV=development BROWSERSLIST_ENV=development BROWSERSLIST_CONFIG=.config/.browserslistrc webpack --config .config/webpack/webpack.dev.js --watch",
    "shopify:serve": "cd shopify && shopify theme serve",
    ...
  }
}
```
